app/controllers/welcome_controller.rb

```diff
@@ -1,10 +1,9 @@
 class WelcomeController < ApplicationController
   def index
     # TODO: Many '@', improvement is need
-    @members    = User.order(:name)
-    not_happened_event = Event.not_happened.first
-    unless not_happened_event.nil?
-      @next_event = EventDecorator.new(not_happened_event, view_context)
-    end
+    @members = User.order(:name)
+    next_event = Event.not_happened.first
+
+    @next_event = EventDecorator.new(next_event, view_context) if next_event
```

Comment:
Yeah I think it too, but is the best name I can figure out about this context... Can you send me some thoughts about it?

Or we can keep like that, until we made the refactor proposed by [line 3](https://github.com/guru-pr/gurupr/pull/145/files/cffa0a026fe18d4ba3c9f72308dd03f23eee98fd#diff-0cfca03f12bc77e6ad10935988e0f304R3)